app/controllers/Application.java

```diff
@@ -1490,7 +1500,61 @@ public static Result restFlowMetricsGraphData(String flowDefId) {
       datasets.add(dataset);
     }
 
-    return ok(new Gson().toJson(datasets));
+    JsonArray sortedDatasets = sortJsonArray(datasets);
```

Comment:
This refactoring looks good but I would put it separately from adding Langoustine support to Dr-Elephant.